.github/workflows/ci.yml

```diff
@@ -21,6 +18,5 @@ jobs:
       contents: write
     steps:
       - uses: fastify/github-action-merge-dependabot@v3
-        if: ${{ github.actor == 'dependabot[bot]' && github.event_name == 'pull_request' }}
-        with:
-          github-token: ${{ secrets.GITHUB_TOKEN }}
+        if: ${{ github.actor == 'dependabot[bot]' && github.event_name == 'pull_request'
+          }}
```

Comment:
@marco-ippolito not sure this is a safe change, in any case this line (the if: ...) can be entirely removed